pkg/s3select/json/data/1.json

```diff
@@ -0,0 +1,250 @@
+{"id":"8819736894"}
+{"id":"8819736896"}
+{"id":"8819736897"}
+{"id":"8819736899"}
+{"id":"8819736900"}
+{"id":"8819736901"}
+{"id":"8819736903"}
+{"id":"8819736904"}
+{"id":"8819736909"}
+{"id":"8819736910"}
+{"id":"8819736911"}
+{"id":"8819736912"}
+{"id":"8819736915"}
+{"id":"8819736917"}
+{"id":"8819736919"}
+{"id":"8819737264"}
+{"id":"8819737265"}
+{"id":"8819737266"}
+{"id":"8819737267"}
+{"id":"8819737268"}
+{"id":"8819737269"}
+{"id":"8819737273"}
+{"id":"8819737278"}
+{"id":"8819737279"}
+{"id":"8819737281"}
+{"id":"8819737282"}
+{"id":"8819737284"}
+{"id":"8819737285"}
+{"id":"8819737286"}
+{"id":"8819737288"}
+{"id":"8819737289"}
+{"id":"8819737304"}
+{"id":"8819737307"}
+{"id":"8819737308"}
+{"id":"8819737309"}
+{"id":"8819737312"}
+{"id":"8819737314"}
+{"id":"8819737315"}
+{"id":"8819737319"}
+{"id":"8819737320"}
+{"id":"8819737321"}
+{"id":"8819737323"}
+{"id":"8819737326"}
+{"id":"8819737331"}
+{"id":"8819737333"}
+{"id":"8819737334"}
+{"id":"8819737335"}
+{"id":"8819737337"}
+{"id":"8819737339"}
+{"id":"8819737343"}
+{"id":"8819737345"}
+{"id":"8819737346"}
+{"id":"8819737349"}
+{"id":"8819737350"}
+{"id":"8819737351"}
+{"id":"8819737352"}
+{"id":"8819737353"}
+{"id":"8819737357"}
+{"id":"8819737358"}
+{"id":"8819737362"}
+{"id":"8819737363"}
+{"id":"8819737368"}
+{"id":"8819737373"}
+{"id":"8819737374"}
+{"id":"8819737378"}
+{"id":"8819737379"}
+{"id":"8819737380"}
+{"id":"8819737386"}
+{"id":"8819737388"}
+{"id":"8819737391"}
+{"id":"8819737395"}
+{"id":"8819737396"}
+{"id":"8819737398"}
+{"id":"8819737400"}
+{"id":"8819737403"}
+{"id":"8819737404"}
+{"id":"8819737407"}
+{"id":"8819737408"}
+{"id":"8819737409"}
+{"id":"8819737411"}
+{"id":"8819737412"}
+{"id":"8819737414"}
+{"id":"8819737416"}
+{"id":"8819737417"}
+{"id":"8819737418"}
+{"id":"8819737421"}
+{"id":"8819737423"}
+{"id":"8819737425"}
+{"id":"8819737428"}
+{"id":"8819737432"}
+{"id":"8819737434"}
+{"id":"8819737435"}
+{"id":"8819737438"}
+{"id":"8819737439"}
+{"id":"8819737441"}
+{"id":"8819737442"}
+{"id":"8819737446"}
+{"id":"8819737448"}
+{"id":"8819737452"}
+{"id":"8819737454"}
+{"id":"8819737455"}
+{"id":"8819737456"}
+{"id":"8819737458"}
+{"id":"8819737460"}
+{"id":"8819737461"}
+{"id":"8819737462"}
+{"id":"8819737470"}
+{"id":"8819737475"}
+{"id":"8819737476"}
+{"id":"8819737479"}
+{"id":"8819737481"}
+{"id":"8819737482"}
+{"id":"8819737483"}
+{"id":"8819737484"}
+{"id":"8819737485"}
+{"id":"8819737486"}
+{"id":"8819737488"}
+{"id":"8819737489"}
+{"id":"8819737490"}
+{"id":"8819737491"}
+{"id":"8819737492"}
+{"id":"8819737494"}
+{"id":"8819737496"}
+{"id":"8819737501"}
+{"id":"8819737504"}
+{"id":"8819737508"}
+{"id":"8819737509"}
+{"id":"8819737511"}
+{"id":"8819737512"}
+{"id":"8819737515"}
+{"id":"8819737516"}
+{"id":"8819737517"}
+{"id":"8819737519"}
+{"id":"8819737520"}
+{"id":"8819737522"}
+{"id":"8819737525"}
+{"id":"8819737528"}
+{"id":"8819737531"}
+{"id":"8819737533"}
+{"id":"8819737535"}
+{"id":"8819737537"}
+{"id":"8819737539"}
```

Comment:
No 250 is to address a bug, we would only list the first 20 lines in master for this json.

Reply:
Hmm, ok, but having a long file does not really help as it does not prove that the bug is fixed on even longer files. I do not understand the problem with the original code, but it may be possible to create a minimal test case. Even if not, it is better to simply write tests for the requirement and implementation rather than an older bug. Over time, this file is just dead-weight, as we won't really care about old bugs after a correct implementation emerges.

Reply:
Another point is that if it really is needed to test with such a long file, a few lines of code can generate this content quite easily.

Reply:
> Hmm, ok, but having a long file does not really help as it does not prove that the bug is fixed on even longer files. I do not understand the problem with the original code, but it may be possible to create a minimal test case. Even if not, it is better to simply write tests for the requirement and implementation rather than an older bug. Over time, this file is just dead-weight, as we won't really care about old bugs after a correct implementation emerges.

We are not even using the original code. I can simply remove this file as its not really needed, we have other files with obscure looking JSON. 

> Another point is that if it really is needed to test with such a long file, a few lines of code can generate this content quite easily.

I don't like to generate files for this, it's just good to see actual JSON files and use them when reproducing issues - it is perfectly alright to keep these files since these are variations which are rare to find. 

I will remove the 250 line file as it's not needed with the library anymore.